identifiers/country-eu/README.md

```diff
@@ -0,0 +1,12 @@
+# Open Civic Data Divisions: European Union
+
+This folder list the following administrative divisions of the European Union:
+
+* Member state of the European Union [ref](https://en.wikipedia.org/wiki/Member_state_of_the_European_Union).
+* European Parliament constituency [ref](https://en.wikipedia.org/wiki/European_Parliament_constituency).
+
+## Types
+
+* `region`: Region
+* `country`: Country
+* `ed`: electoral district
```

Comment:
Let's use `eup` for the European Parliament electoral districts, otherwise these could conflict with `ed` use within a country's OCD IDs

Reply:
Done.